### PR TITLE
ui/workspace/ - update attribute name styles throughout the app #LMR-701

### PR DESCRIPTION
**Pull request by justmesr**
Default attribute styles added.
Unified document view attribute and value styles.
Smart-doc attribute styles were left unchanged.

Signed-off-by: justmesr <simon.kocurek@gmail.com>

### CODE REVIEW

File: src/app/view/perspectives/post-it/document/post-it-document.component.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
               (keydown.enter)="selectionHelper.setEditMode(true); selectionHelper.focus()"
               (click)="clickOnAttributePair(0, row)">
 
-          <input class="border-0"
+          <input class="border-0 {{ isDefaultAttribute(attributePair.attribute) ? 'default-attribute' : null }}"
```
Comment (kubedo8) [inline src/app/view/perspectives/post-it/document/post-it-document.component.html]: Isn't better to use it like `[class.default-attribute]="isDefaultAttribute(attributePair.attribute)"` ?

Comment (justmesr) [reply src/app/view/perspectives/post-it/document/post-it-document.component.html]: I don't know. Are there any advantages?

If not, we can just choose one and stick to that. As far as I know, we use the `{{ }}` syntax more.

File: src/app/view/perspectives/search/documents/search-documents.component.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
+@import "../../../../shared/variables";
+
+/deep/.search-documents-attribute {
```
Comment (livthomas) [inline src/app/view/perspectives/search/documents/search-documents.component.scss]: I think we should avoid using `deep` selector. Why does it need to be used here?

File: src/app/view/perspectives/post-it/document/post-it-document.component.html (hunk 1 above)
Comment (livthomas) [reply src/app/view/perspectives/post-it/document/post-it-document.component.html]: I think it does not really matter although I find the syntax suggested by @kubedo8 much cleaner. We use both styles across the application.

File: src/app/view/perspectives/post-it/document/post-it-document.component.scss [hunk 3]
```diff
@@ -64,8 +64,15 @@
...
-    width: 80px;
+    width: 75px;
+    color: $primary;
+    font-style: italic;
+  }
+
+  .default-attribute input {
+    font-style: normal;
+    font-weight: bold;
+    color: #000000;
```
Comment (livthomas) [inline src/app/view/perspectives/post-it/document/post-it-document.component.scss]: I am not really sure if this is what @marvec meant. I think all attributes should be in italics and the default one in bold as well. But I might be wrong.

If I am right you don't really need to define some special class here and can just use:
```html
[class.font-weight-bold]="isDefaultAttribute(attributePair.attribute)"
```

File: src/app/view/perspectives/post-it/document/post-it-document.component.scss [hunk 4]
```diff
@@ -76,6 +83,7 @@
     padding: 0 3px;
     width: 100%;
     pointer-events: none;
+    color: #000000;
```
Comment (livthomas) [inline src/app/view/perspectives/post-it/document/post-it-document.component.scss]: I am not sure if we really want to use pure black color anywhere in our application.

File: src/app/view/perspectives/search/documents/search-documents.component.scss (hunk 2 above)
Comment (justmesr) [reply src/app/view/perspectives/search/documents/search-documents.component.scss]: html is pasted in as a string.
Angular ignores the class unless the _ngcontext matches.

Comment (livthomas) [reply src/app/view/perspectives/search/documents/search-documents.component.scss]: And why do we put there HTML from string? Cannot it be directly in the template?

File: src/app/view/perspectives/post-it/document/post-it-document.component.scss (hunk 3 above)
Comment (justmesr) [reply src/app/view/perspectives/post-it/document/post-it-document.component.scss]: I've tried that. There was no way to tell which one is the default one, so I made it more visible.

File: src/app/view/perspectives/search/documents/search-documents.component.scss (hunk 2 above)
Comment (justmesr) [reply src/app/view/perspectives/search/documents/search-documents.component.scss]: Probably. (I guess the recursive nature of some values was the reason @kubedo8 made it like that)
It is out of the scope of this PR, so I'll leave at that.

File: src/app/view/perspectives/post-it/document/post-it-document.component.scss (hunk 4 above)
Comment (justmesr) [reply src/app/view/perspectives/post-it/document/post-it-document.component.scss]: We do. It's the default for text in inputs.

File: src/app/view/perspectives/search/documents/search-documents.component.scss (hunk 2 above)
Comment (kubedo8) [reply src/app/view/perspectives/search/documents/search-documents.component.scss]: @justmesr yes you are right. Document may have nested attributes and values and generating html directly in the template was very slow even for document with few attributes.

**Pull request merged**